.github/workflows/ci.yml

```
@@ -0,0 +1,50 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+      - master
+      - "*-stable"
+  pull_request:
+    branches:
+      - master
+      - "*-stable"
+
+jobs:
+  ci:
+    name: "Run Tests (${{ matrix.label }})"
+    runs-on: "ubuntu-latest"
+    env:
+      # See https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby#matrix-of-gemfiles
+      BUNDLE_GEMFILE: ${{ matrix.gemfile }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - label: Ruby 2.7
+            ruby_version: "2.7"
+            gemfile: Gemfile
+          - label: Ruby 3.0
+            ruby_version: "3.0"
+            gemfile: Gemfile
+          - label: Ruby 3.1
+            ruby_version: "3.1"
+            gemfile: Gemfile
+          - label: JRuby 9.3.4.0
+            ruby_version: "jruby-9.3.4.0"
+            gemfile: Gemfile-jruby
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+      - name: "Set up ${{ matrix.label }}"
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby_version }}
+          bundler-cache: true
+      - name: Run Minitest based tests
+        run: bash script/test
```

If you'd like, you can remove `bash` here
```suggestion
        run: script/test
```